webhook-bridge/svix-webhook-bridge-plugin-webhook-receiver/src/lib.rs

```diff
@@ -104,3 +145,35 @@ async fn route(
         http::StatusCode::NOT_FOUND
     }
 }
+
+/// Attempts to run the payload through a js transformation.
+async fn transform(
+    payload: JsObject,
+    script: String,
+    tx: Option<TransformerTx>,
+) -> Result<JsObject, http::StatusCode> {
+    if let Some(transformer_tx) = tx {
```

Comment:
This is definitely nitpicky, but how do you feel about restructuring this to something like

```
let transformer_tx = if let Some(tx) {
   tx
} else {
   tracing::error!("transformations are not available");
   return Err(http::StatusCode::INTERNAL_SERVER_ERROR)
}
// ... the rest
```

This way, the core business logic isn't as nested.

Reply:
It's a great note, but I did it a little different.

I think the flow can be improved with a refactor of the handler (to return Result instead of StatusCode). Ideally, this function would accept a `TransformerTx` instead of an `Option<_>`.

If the config polish PR ends up moving the HTTP server code into the bridge binary, I'll look to fix this up more better during that work.